pkg/maps/ctmap/per_cluster_ctmap.go

```diff
@@ -124,17 +122,17 @@ func (v *PerClusterCTMapVal) String() string    { return fmt.Sprintf("fd=%d", v.
 func (v *PerClusterCTMapVal) New() bpf.MapValue { return &PerClusterCTMapVal{} }
 
 // NewPerClusterCTMaps returns a new instance of the per-cluster CT maps manager.
-func NewPerClusterCTMaps(ipv4, ipv6 bool) *perClusterCTMaps {
+func NewPerClusterCTMaps(ipv4, ipv6 bool, outerMapEntries int) *perClusterCTMaps {
```

Comment:
Sorry, I know that it was me who suggested to make the number of entries a parameter, but upon a more detailed look it is probably better to hardcode it. The reason is that `cmtypes.ClusterIDMax` is already accessed directly once in `cleanup`, and indirectly by `ValidateClusterID`, so that would increase the risk of inconsistencies. The other alternative would be to also refactor the other usages, but that looks out of scope here.